gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/ResultPage.java

```diff
@@ -52,10 +50,6 @@ public String getLimitsQueryParam() {
     }
 
     private static String toQueryParam(final List<Integer> list) {
-        try {
-            return UriUtils.encode(list.stream().map(String::valueOf).collect(joining(",")), "UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            throw new IllegalStateException("Missing UTF-8 charset");
-        }
+        return list.stream().map(String::valueOf).collect(joining("%2C"));
```

Comment:
What if there's some special char in some value in the list?

Reply:
what kind of special char can be created by conversion from `Integer`? Show me that number pls ;-)